packages/hub/src/app/models/[owner]/[slug]/EditSquiggleSnippetModel.tsx

```diff
@@ -262,6 +264,87 @@ export const EditSquiggleSnippetModel: FC<Props> = ({
 
   const { height, ref } = useAvailableHeight();
 
+  const checkedVersion = useValidSquiggleVersion(version);
+
+  const playgroundProps: Parameters<typeof VersionedSquigglePlayground>[0] = {
```

Comment:
Maybe leave a comment explaining this type def a bit, and/or why you didn't just include the whole of          <VersionedSquigglePlayground> here.

Reply:
`Parameters<typeof MyComponent>[0]` is pretty standard and it'd be weird to explain it here but not in other places.

Ideally, it should be `ExtractProps<typeof MyComponent>`, but we don't have a shared place where we could store that type helper.

Instead of doing this, I added a comment explaining why props need to be built separately from `<VersionedSquigglePlayground>` JSX call.